### PR TITLE
 soc: arm: stm32l0/stm32l1: set voltage scaling to range1

### PR DESCRIPTION
**Pull request by ABOSTM**
soc: arm: stm32l0/stm32l0: set voltage scaling to range1

Default Voltage scaling range selection (range2)
doesn't allow to configure Max frequency
switch to range1 to match any frequency

Applicable to both stm32l0 and stm32l1

### CODE REVIEW

**Review by erwango — APPROVED** [review]

**Comment by tagunil** [comment]
Shouldn't such things be a part of the clock control driver?

File: soc/arm/st_stm32/stm32l1/soc.c [hunk 1]
```diff
@@ -15,6 +15,9 @@
 #include <arch/arm/aarch32/cortex_m/cmsis.h>
 #include <linker/linker-defs.h>
 #include <string.h>
+#include <stm32_ll_bus.h>
+#include <stm32_ll_pwr.h>
+
```
Comment (tagunil) [inline soc/arm/st_stm32/stm32l1/soc.c:20]: Looks like an excessive empty line here.

Comment (ABOSTM) [reply soc/arm/st_stm32/stm32l1/soc.c:20]: Thanks . Fixed.

**Comment by ABOSTM** [comment]
> Shouldn't such things be a part of the clock control driver?

That was an alternative we discussed with @erwango .
Knowing that it is done that way (in soc) for STM32H7 and STM32L5, we decided to continue that way.

**Comment by erwango** [comment]
> That was an alternative we discussed with @erwango .
> Knowing that it is done that way (in soc) for STM32H7 and STM32L5, we decided to continue that way.

Indeed, and since till now there is no support to change it, either through static board configuration or dynamically at run time, I don't think it is justified to move this in clock_control driver.

**Review by str4t0m — APPROVED** [review]

**Pull request merged**